modules/core/packer/roles/install-consul-template/files/run-consul-template.sh

```diff
@@ -307,6 +307,12 @@ template {
   create_dest_dirs = true
   error_on_missing_key = true
   perms = 0600
+
+  # Vault Tokens should not change often. We can set this to something pretty long.
```

Comment:
@guangie88 Gotta update this too https://github.com/GovTechSG/amazon-linux-consul/blob/master/packer/roles/install-consul-template/files/run-consul-template.sh#L318

Reply:
Please see also https://github.com/GovTechSG/amazon-linux-consul/pull/3